packages/button/README.md

```diff
@@ -55,18 +55,20 @@ import Button from '@leafygreen-ui/button';
+| `darkMode`   | `boolean`                                                        | Determines if the component renders in dark mode                                                                                                      | `false`     |
+| `size`       | `'xsmall'` \| `'small'` \| `'normal'` \| `'large'`               | Sets the size variant of the button.                                                                                                                  | `'normal'`  |
+| `children`   | `node`                                                           | The content that will appear inside of the `<Button />` component.                                                                                    | `null`      |
+| `onClick`    | `function`                                                       | The event handler function for the 'onclick' event. Receives the associated `event` object as the first argument.                                     |             |
+| `className`  | `string`                                                         | Adds a className to the class attribute.                                                                                                              | `''`        |
+| `value`      | `string`                                                         | Sets the HTML `value` attribute.                                                                                                                      |             |
+| `disabled`   | `boolean`                                                        | Disabled the button                                                                                                                                   | `false`     |
+| `as`         | `React.ElementType`                                              | Determines the root element. For example, `Link` or `a` tags can be supplied to replace `button` from being the DOM element that wraps the component. | `button`    |
+| `href`       | `string`                                                         | If a href is supplied it will change the `as` value, such that the component renders inside of an `a` tag instead of inside of a `button` tag.        |             |
+| `glyph`      | `React.ReactElement`                                             | When supplied a rendered Icon, renders a button with the specified glyph on the left-side of the content, and adjusts Button padding accordingly.     |             |
+| `forceState` | `{ focused?: boolean, active?: boolean }`                        | Force the component to display the specified interaction states.                                                                                      |             |
```

Comment:
why are we adding this prop?

Reply:
He ran this one by me. There are cases where we need to force a hover state for, say, an active menu triggered by a button.